packages/ui/src/lib/layouts/DetailsPage.svelte

```diff
@@ -0,0 +1,89 @@
+<script lang="ts">
+import { createEventDispatcher } from 'svelte';
+
+import CloseButton from '../button/CloseButton.svelte';
+import Link from '../link/Link.svelte';
+
+export let title: string;
+export let titleDetail: string | undefined = undefined;
+export let subtitle: string | undefined = undefined;
+export let breadcrumbLeftPart: string | undefined = undefined;
+export let breadcrumbRightPart: string | undefined = undefined;
+export let breadcrumbTitle: string | undefined = '';
```

Comment:
I know this is very similar to the FormPage, but this is not really extensible, to have that many property for the breadcrumb,.

Not a blocker for this PR, but we should have a property being an array, to ensure typesafety to one object and not having that many, it would look less ugly and offer more customizaton